src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmissingleadingasterisk/Example1.java

```diff
@@ -14,34 +15,36 @@
  *   int value = 0;
  * </pre>
  */
-class JavaStyle {} // ok
+class JavaStyle {}
 
 /** Valid Scala-style comment.
   * Some description here.
   **/
-class ScalaStyle {} // ok
+class ScalaStyle {}
 
 /** **
   * Asterisks on first and last lines are optional.
   * */
-class Asterisks {} // ok
+class Asterisks {}
 
 /** No asterisks are required for single-line comments. */
-class SingleLine {} // ok
+class SingleLine {}
 
-/** // violation on next blank line, javadoc has lines without leading asterisk.
+// violation 2 lines below
```

Comment:
please place part of message to this line, this Check has several violation messages
https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/cc59ed8_2024024321/checks/javadoc/javadocmissingleadingasterisk.html#Violation_Messages

so user should know what violation is reported.

apply this rule to all changes.

Reply:
Please review if it is all good now